frontend/src/components/QuestionResults.tsx

```diff
@@ -1,16 +1,60 @@
-import { Question } from '../client';
-import { Stack, Typography } from '@mui/material';
+import { Answer, Question } from '../client';
+import { Container, Stack } from '@mui/material';
+import Chart from 'react-apexcharts';
 
 interface QuestionResultsProps {
   question: Question;
 }
 
 export default function QuestionResults(props: QuestionResultsProps) {
+  const series = [];
+  const labels = [];
+
+  // The order of these changes on database changes by voting, to maintain a stable chart, we need to sort the answers by name
+  const answers = props.question.answers.sort(answerSorter);
+
+  for (const answer of answers) {
+    series.push(answer.count);
+    labels.push(answer.text);
```

Comment:
It would be nicer to use `const series = answers.map((answer) => answer.count)` after sorting

Reply:
You certainly have a point here, I will fix that.